src/components/content/deployedServices/myServices/MyServices.tsx

```diff
@@ -172,6 +178,62 @@ function MyServices(): React.JSX.Element {
                     </Button>
                 ),
             },
+            {
+                key: 'scale',
+                label: (
+                    <Button
```

Comment:
both scale and modify button must be placed at the end of the right margin of form button